src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    add: (state, action) => {
+      state.books.push(action.payload);
+    },
+    remove: (state, action) => {
+      state.books = state.books.filter((book) => book.id !== action.payload);
```

Comment:
- [Optional] To follow the naming convention closely, I would suggest updating your reducer methods  name, try to use `addBook` instead of `add` and `removeBook` instead of `remove`

Reply:
Thank @TadesseDev for taking the time to review this branch.
I will rename the reducer methods as you suggested.